src/clients/common/notificationBannerStatic.js

```diff
@@ -0,0 +1,102 @@
+
+/**
+ * In a same:ish way the notification banner and dialogs are handled push child to a container except here make sure theres only one child (for now)
+ */
+/**
+ * 
+ * @param {*} content we are expecting object with a "style" and "message"
+ * @returns 
+ */
+export function showNotificationBannerStatic(content) {
+
+  if (arguments.length === 0 || content === null) {
+    console.log('NotificationBannerStatic needs arguments');
+    return;
+  }
+  if(!content || !content.style || !content.text){
+    console.log('showNotificationBannerStatic argument needs to have object with correct fields, see JSDoc comment');
+    return;
+  }
+
+  getNotificationBannerStaticHtml()
+    .then(html => createNotificationBannerStatic(content, html))
+    .catch(error => console.log('Error while fetching NotificationBannerStatic html: ', error));
+}
+
+async function getNotificationBannerStaticHtml() {
+  const response = await fetch('/../common/templates/notificationBannerStatic.html');
+  const html = await response.text();
+  return html;
+}
+
+function createNotificationBannerStatic(content, html) {
+  const templateElement = getNewNotificationBannerStatic(html);
+  
+  styleBasedOnType();
+  addTextToElement();
+  addElementToContainer();
+
+  //************************************************************************************** */
+  // Helper functions for createNotificationBannerStatic:
+
+  function getNewNotificationBannerStatic(html) {
+    const NotificationBannerStaticDocument = getNotificationBannerStaticDocument();
+    const NotificationBannerStaticTemplate = NotificationBannerStaticDocument.getElementById('notificationBannerStaticTemplate');
+    const NotificationBannerStaticFragment = NotificationBannerStaticTemplate.content.cloneNode(true);
+    const NotificationBannerStaticElement = NotificationBannerStaticFragment.getElementById('notificationBannerStatic');
+    return NotificationBannerStaticElement;
+
+    function getNotificationBannerStaticDocument() {
+      const parser = new DOMParser();
+      const NotificationBannerStaticDocument = parser.parseFromString(html, 'text/html');
+      return NotificationBannerStaticDocument;
+    }
+
+  }
+  function styleBasedOnType(){
+    
+    let backgroundColor = '';
+    let iconColor = '';
+    let icon = '';
+
+    switch (content.style) {
```

Comment:
avoid switch and use style helper function mentioned in `src/clients/common/notificationBanner.js`